core/common/src/test/java/alluxio/PropertyKeyTest.java

```diff
@@ -58,6 +72,12 @@ public void isValid() throws Exception {
   }
 
   @Test
+  public void aliasIsValid() throws Exception {
+    assertTrue(PropertyKey.isValid(mTestProperty.toString()));
+    assertTrue(PropertyKey.isValid(mNullAliasTestProperty.toString()));
```

Comment:
not sure what are we testing here? could you explain this? why this line 77 is related to alias?